requirements.txt

```diff
@@ -1,18 +1,18 @@
-cartopy==0.20.0
-cmocean==2.0
-dask==2021.9.1
-fsspec==2021.10.0
-joblib==1.0.1
-matplotlib==3.4.3
-netCDF4==1.5.7
-numba==0.54.0
-numpy==1.21.2
-osgeo==0.0.1
-pandas==1.3.3
-pyproj==3.2.1
-pyresample==1.21.1
-pytest==6.2.5
-scipy==1.7.1
-seawater==3.3.4
-Shapely==1.7.1
-xarray==0.19.0
+cartopy
+cmocean
+dask
+fsspec
+joblib
+matplotlib
```

Comment:
@pgierz please note we have some requirements pinned to some versions in `requirements-py37.yml` :)